pkg/accelerator/accelDeviceProvider.go

```diff
@@ -124,6 +125,17 @@ func (ap *accelDeviceProvider) GetFilteredDevices(devices []types.PciDevice, rc
 		}
 	}
 
+	if af.VfOnly {
+		vfOnlyDevices := make([]types.PciDevice, 0)
+		for _, dev := range filteredDevice {
+			if !utils.IsSriovPF(dev.GetPciAddr()) {
+				vfOnlyDevices = append(vfOnlyDevices, dev)
+			}
+		}
+
+		filteredDevice = vfOnlyDevices
+	}
+
```

Comment:
Would it make sense to have some kind of DeviceSelector for this as well?

Reply:
I try to use it first but the problem is the `DeviceSelector` use a list of string in the interface. that  is why I follow the same case as [https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin/pull/408/files/62867e6214b9d1aa45b78ce19144e6785bbf8b88#diff-640d0436ec4d8e1927e54b82a9d791bfb0127eff373c073edc53f69627b4b43aR202]